### ISSUE TITLE
Remove json format

### ISSUE THREAD

**Issue by nichtich** [mixu/gr#25]
Support of the `--json` output format spoils the source code at many places adding complexity, bad design, and bugs. I'd rather remove the feature. Otherwise gr needs a major code refactoring to abstract from output formats, making it even more complex.

**Comment by mixu** [comment]
Makes sense to me, it was one of those features that I thought might turn out to be useful when building out gr, but at least personally I haven't found a really compelling use case for it.
